src/EnergyPlus/HighTempRadiantSystem.cc

```diff
@@ -451,6 +458,12 @@ namespace HighTempRadiantSystem {
             }
 
             if (state.dataIPShortCut->lAlphaFieldBlanks(7)) {
+                ShowSevereEmptyField(state,
+                                     eoh,
+                                     state.dataIPShortCut->cAlphaFieldNames(7),
+                                     state.dataIPShortCut->cAlphaFieldNames(6),
+                                     state.dataIPShortCut->cAlphaArgs(6));
+                ErrorsFound = true;
             } else if ((highTempRadSys.setptSched = Sched::GetSchedule(state, state.dataIPShortCut->cAlphaArgs(7))) == nullptr) {
                 ShowSevereItemNotFound(state, eoh, state.dataIPShortCut->cAlphaFieldNames(7), state.dataIPShortCut->cAlphaArgs(7));
                 ErrorsFound = true;
```

Comment:
A7 is now a required field so it can't be blank (I think) but it can have an invalid name, so 467 should be the only conditional?